packages/@aws-cdk/integ-tests/lib/test-case.ts

```diff
@@ -21,18 +21,46 @@ export interface IntegTestCaseProps extends TestOptions {
  * apply to all stacks under this case.
  */
 export class IntegTestCase extends Construct {
-  constructor(scope: Construct, id: string, props: IntegTestCaseProps) {
+  constructor(scope: Construct, private readonly id: string, private readonly props: IntegTestCaseProps) {
     super(scope, id);
+  }
+
+  /**
+   * The integration test manifest for this test case. Manifests are used
+   * by the integration test runner.
+   */
+  get manifest(): IntegManifest {
+    return {
+      version: '',
```

Comment:
I think we need to use a valid semver version here. We will get errors when trying to read the manifest if this does not have a valid version. We will also probably need this once we go live so we can add new features that depend on certain versions.